jobsapp/views/home.py

```diff
@@ -71,3 +76,21 @@ def get(self, request, *args, **kwargs):
             raise Http404(_("Job doesn't exists"))
         context = self.get_context_data(object=self.object)
         return self.render_to_response(context)
+
+
+class ContactView(FormView):
+    template_name = "contact_us.html"
+    form_class = ContactForm
+    success_url = "/"
+
+    def post(self, request: HttpRequest, *args: str, **kwargs: Any) -> Union[HttpResponse, Any]:
+        form = self.get_form()
+        if form.is_valid():
+            messages.info(self.request, _("Mensage sent successfully"))
+            return self.form_valid(form)
+        messages.warning(self.request, _("Mensage not sent"))
+        return super().post(request, *args, **kwargs)
+
+    def form_valid(self, form):
+        form.send_email()
```

Comment:
You can put `message.info` or `message.success` here.

Reply:
It's rigth, makes more sense. Thanks :+1: